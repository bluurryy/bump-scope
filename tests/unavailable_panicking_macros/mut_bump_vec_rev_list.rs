use bump_scope::{Bump, MutBumpVecRev, mut_bump_vec_rev};

pub fn one_two_three<'a>(bump: &'a mut Bump, two: i32) -> MutBumpVecRev<i32, &'a mut Bump> {
    mut_bump_vec_rev![in bump; 1, two, 3]
}

fn main() {}
