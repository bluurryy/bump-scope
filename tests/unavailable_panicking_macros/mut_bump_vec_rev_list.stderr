error: the potentially panicking api of `mut_bump_vec_rev!` is not available
       help: enable `bump-scope`'s "panic-on-alloc" feature or use `try`:
       `mut_bump_vec_rev![try in bump; 1, two, 3]`
 --> tests/unavailable_panicking_macros/mut_bump_vec_rev_list.rs:4:5
  |
4 |     mut_bump_vec_rev![in bump; 1, two, 3]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `$crate::__mut_bump_vec_rev_panic_on_alloc` which comes from the expansion of the macro `mut_bump_vec_rev` (in Nightly builds, run with -Z macro-backtrace for more info)
